docs/cpu_templates/cpu-templates.md

```diff
@@ -175,11 +179,14 @@ curl --unix-socket /tmp/firecracker.socket -i  \
       }'
 ```
 
-This CPU templates will do the following with the ARM register `0x603000000013c020`:
+This CPU templates will do the following:
 
-- will clear bits `0b0000000000001111000000000000111100000000000000000000000000000000`
-- will leave bits `0b1111111111110000111111111111000011111111111111111111111111111111`
-  intact.
+- add checks for KVM capabilities: KVM_CAP_ARM_PTRAUTH_ADDRESS and KVM_CAP_ARM_PTRAUTH_GENERIC
```

Comment:
Maybe a sentence or two on why this is useful and links to kernel docs/source to get the values from?

Reply:
added more info